labellab-client/src/components/project/models.js

```diff
@@ -0,0 +1,150 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { Link } from 'react-router-dom'
+import {
+  Button,
+  Container,
+  Dropdown,
+  Header,
+  Table,
+  Modal
+} from 'semantic-ui-react'
+import PropTypes from 'prop-types'
+
+import { fetchProject } from '../../actions'
+import './css/models.css'
+
+class ModelsIndex extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      modelType: "classifier",
+      open: false,
+    }
+
+    this.model = [
+      {
+        key: 'classifier',
+        text: 'Classifier',
+        value: 'classifier'
+      }
+    ]
+  }
+
+  fetchProjectCallback = () => {
+    const { fetchProject, project } = this.props
+    fetchProject(project.projectId)
+  }
+
+  getModelType = (event, {value}) => {
+    this.setState({
+      modelType: value
+    })
+  }
+
+  handleClose = () => {
+    this.setState({
+      open: false
+    })
+  }
+
+  handleOpen = () => {
+    this.setState({
+      open: true
+    })
+  }
```

Comment:
You can handle toggle of open state with only one function. Just pass the value you want to update.

Reply:
Sure, made the change